docs/providers/aws/guide/layers.md

```diff
@@ -36,6 +36,7 @@ layers:
     licenseInfo: GPLv3 # optional, a string specifying license information
     allowedAccounts: # optional, a list of AWS account IDs allowed to access this layer.
       - '*'
+    retain: false # optional, false by default. If true, layer versions are not deleted as new ones are created
```

Comment:
Should we also add this config to the `serverless.yml.md` file? (the root `layers` definition is completely missing right now).